pkg/controllers/dynakube/otelc/secret/conditions.go

```diff
@@ -0,0 +1,5 @@
+package secret
+
+const (
+	secretConditionType = "TelemetryApiCredentialsSecretCondition"
```

Comment:
Let's be consistent with the naming of the feature
```suggestion
	secretConditionType = "TelemetryServiceApiCredentialsSecretCondition"
```

Reply:
renamed in 0bac0b98d